### ISSUE TITLE
No coverage information found for ...

### ISSUE THREAD

**Issue by bjornforsberg** [dewski/vscode-simplecov#4]
Thanks very much for building this! 😊

Trying this out on our Rails 7 app and latest VS Code, but unable to get it working. I have the coverage folder with `.resultset.json` but the VS Code extension never seems to match files, even when the paths exist in the results.

It always shows "No coverage information found for xxxxx"

Here's a screenshot, where the 2nd item in the resultset.json should match the currently opened file:
![CleanShot 2024-01-29 at 17 10 52](https://github.com/dewski/vscode-simplecov/assets/1411883/4eee7479-f62a-4cb3-acc3-3ae1db7a73a7)

Any ideas what I'm doing wrong here?

Here's my simplecov block in case it's useful:

```
require "simplecov"

SimpleCov.start "rails" do
  command_name "test"
  add_group("Services", "app/services")
  add_group "Components", "app/components"
  add_filter %w[
    app/views
    bin
    coverage
    log
    test
    vendor
    node_modules
    db
    public
    storage
    tmp
  ]
end
```

And here's the full info in `.last_run.json`
```
{
  "result": {
    "line": 50.6
  }
}
```

**Comment by dewski** [comment]
Thanks for your detailed bug report! Providing the JSON from your screenshot was very helpful in reproducing the bug you reported. I went ahead and fixed the bug in #5 and released [v0.0.3](https://marketplace.visualstudio.com/items?itemName=dewski.simplecov) on the VS Code Marketplace. Please feel free to re-open the issue if this did not address what you encountered.

**Comment by bjornforsberg** [comment]
@dewski - Thanks so much for this Garrett, glad to hear it was a useful report 😊 Will give it another shot tomorrow, but does look like it should fix my issue. Appreciate the quick help 🙏
